tests/manage/z_cluster/test_delete_osd_deployment.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import pytest
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier4,
+    ignore_leftovers,
+    skipif_ocs_version,
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.pod import get_osd_deployments
+from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import ceph_health_check
+
+logger = logging.getLogger(__name__)
+
+
+@tier4
+@ignore_leftovers
```

Comment:
may i know why are we using this ??

Reply:
Used it for testing the code. Removed it now